drivers/wifi/nrf700x/Kconfig

```diff
@@ -374,3 +374,26 @@ config NRF_WIFI_BEAMFORMING
 config WIFI_NRF700X_SCAN_TIMEOUT_S
 	int "Scan timeout in seconds"
 	default 30
+
+config NRF_WIFI_2G_BAND
+	bool ""
```

Comment:
Why are these options hidden?

Reply:
The band will be set based on boards. nrf7001 only supports 2.4GHz so directly only 2.4GHz band will be set.

Reply:
We already have that, I thought the purpose of Kconfig option is to choose independent of SoC, no? i.e., even for 7002 we can choose the band.